mne/epochs.py

```diff
@@ -526,6 +516,28 @@ def __init__(self, info, data, events, event_id=None, tmin=-0.2, tmax=0.5,
         self._raw_times = np.arange(start_idx,
                                     int(round(tmax * sfreq)) + 1) / sfreq
         self._set_times(self._raw_times)
+
+        # check reject_tmin and reject_tmax
+        if (reject_tmin is not None) and (np.isclose(reject_tmin, tmin)):
+            # potentially adjust for small deviations due to sampling freq
+            reject_tmin = self.tmin
+        elif (reject_tmin is not None) and (reject_tmin < tmin):
+            raise ValueError("reject_tmin needs to be None or >= tmin")
```

Comment:
Simpler control flow
```suggestion
        if reject_tmin is not None:
            if np.isclose(reject_tmin, tmin):
                # potentially adjust for small deviations due to sampling freq
                reject_tmin = self.tmin
            elif reject_tmin < tmin:
                raise ValueError("reject_tmin needs to be None or >= tmin")
```

Reply:
Addressed in the latest commit